init.sh

```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env bash
 
-echo "📦 Set up a python application"
-docker compose up -d app
\ No newline at end of file
+if [ -f ".env" ]; then
+  echo "🌎 .env exists. Leaving alone"
+else
+  echo "🌎 .env does not exist. Copying .env-example to .env"
+  cp env.example .env
+  YOUR_UID=`id -u`
+  YOUR_GID=`id -g`
```

Comment:
I see this issue when I run the script init.sh. Should the values specify by default in the compose.yml file be used?

<img width="752" alt="image" src="https://github.com/mlibrary/python-starter/assets/47088146/10bfd6db-5b2d-404e-b6ef-7ebd42582635">